tests/integrations/django/test_basic.py

```diff
@@ -40,6 +40,50 @@ def test_view_exceptions(sentry_init, client, capture_exceptions, capture_events
     assert event["exception"]["values"][0]["mechanism"]["type"] == "django"
 
 
+def test_ensures_x_forwarded_header_is_honored_in_sdk_when_enabled_in_django(
+    sentry_init, client, capture_exceptions, capture_events
+):
+    """
+    Test that ensures if django settings.USE_X_FORWARDED_HOST is set to True
+    then the SDK sets the request url to the `HTTP_X_FORWARDED_FOR`
+    """
+    from django.conf import settings
+
+    settings.USE_X_FORWARDED_HOST = True
```

Comment:
Instead of importing settings I suggest using the settings pytest fixture, then your change is automatically reverted at the end of the test

```
def test_foo(settings):
    settings.USE_X_FORWARDED_HOST = True
    ...
```